content/documentation/concepts/nats-queueing.md

```diff
@@ -11,10 +11,11 @@ category = "concepts"
 
 # NATS Queueing
 
-NATS supports message queueing using point-to-point (one-to-one) communication.
+NATS supports message queueing using point-to-point (one-to-one) communication. Although NATS at it's core is 'fire and forget', message queueing can be used as a performance optimizer.
```

Comment:
I'd rename the Title to "Queue Subscribers", and would strike this sentence to instead say "NATS provides a load balancing feature called queue subscriptions.  Using queue subscribers will load balance message delivery across a group of subscribers which can be used to provide application fault tolerance and scale workload processing."

Reply:
resolved